ref/net46/Microsoft.Build.Framework/Microsoft.Build.Framework.cs

```diff
@@ -547,22 +547,31 @@ public sealed partial class TelemetryEventArgs : Microsoft.Build.Framework.Build
 }
 namespace Microsoft.Build.Framework.Profiler
 {
+    public static partial class EvaluationIdProvider
```

Comment:
Can you make this private? I don't see a reason it being public.

Reply:
Made internal

---
In reply to: [157260091](https://github.com/Microsoft/msbuild/pull/2813#discussion_r157260091) [](ancestors = 157260091)